docs/_advanced/receiver.md

```diff
@@ -20,6 +23,47 @@ A receiver is responsible for handling and parsing any incoming requests from Sl
 To use a custom receiver, you can pass it into the constructor when initializing your Bolt for JavaScript app. Here is what a basic custom receiver might look like.
 
 For a more in-depth look at a receiver, [read the source code for the built-in `ExpressReceiver`](https://github.com/slackapi/bolt-js/blob/master/src/receivers/ExpressReceiver.ts)
+
+---
+
+#### Customizing built-in receivers
+
+The built-in `HTTPReceiver`, `ExpressReceiver`, and `SocketModeReceiver` accept several configuration options. For a full list of options, see the [Receiver options reference](/bolt-js/reference#receiver-options).
+
+##### Extracting custom properties
+
+Use the `customPropertiesExtractor` option to extract custom properties from requests. This is particularly useful for extracting HTTP headers that you want to propagate to other services, for example, if you need to propagate a header for distributed tracing.
```

Comment:
Maybe we should add a note here that, depending on what kind of receiver you are building, or which pre-existing receiver you are extending with `customPropertiesExtractor`, the "from requests" part may not be exactly technically correct. For HTTP-based receivers, they would indeed be an HTTP request, but for socket-mode-based receivers, the argument to the extractor would be a websocket message sent from the server.

Reply:
Checking examples/custom-properties code should be helplful to understand the difference between the two (HTTP vs Socket Mode)